### PR TITLE
Search by user entered year feature added

### PR DESCRIPTION
**Pull request by tmsreekanth98**
Solves issue #4 

Changes done:
* Added a new driver menu which allows user to download images by either entering a year (newly added feature) or to download all images (old method).
* Added code to implement the feature
* Proper error messages are displayed if the year is not available in Amul hits website

Screenshot
amul

### CODE REVIEW

File: amul_topicals.py [hunk 1]
```diff
@@ -131,11 +130,44 @@ def open_folder(location):
...
+                break
+                
+        if year_available:
+            try:
+                year_page = fetch_url(BASE_URL+"?s="+str(year_input))
+                per_year = get_year_topicals(year_page, location)
+            except:
+                print("\nUnknown Error occured! Kindly check network connection.")
+                exit(0)
+        else:
```
[the remark this answers is not in the document]
Comment (tmsreekanth98) [reply amul_topicals.py]: Yes. I wanted to add that but forgot at the end. My bad.

[the remark this answers is not in the document]
Comment (tmsreekanth98) [reply amul_topicals.py]: I will quickly add that as well

File: amul_topicals.py [hunk 2]
```diff
@@ -131,11 +130,44 @@ def open_folder(location):
     #make new folder called 'Amul Topicals' in the directory; files will be stored in this folder
     location = make_folder_in_directory(location, 'Amul Topicals')
 
-    for name, url in sorted(year_urls.items()):
-        year_page = fetch_url(BASE_URL + url)
-        per_year = get_year_topicals(year_page,location)
-        d[url[3:]] = per_year
-    json_data = json.dumps(d, sort_keys=True, indent=4, separators=(',', ': '))
+    # Take users choice of the form of download required.
+    print("How would you like to download the topicals?\n1.Particular Year Topicals Only\n2.All Topicals Yearwise\n")
```
[the remark this answers is not in the document]
Comment (tmsreekanth98) [reply amul_topicals.py]: Ok. I will check if i can use that function.

File: amul_topicals.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (vineetjc) [reply amul_topicals.py]: @tmsreekanth98 What about when the user wants to download more than one year? Should we run the program that many times or ask the user if they want to download more till they exit the program themselves? I will leave this to you to make a decision on, let me know what you think.

[the remark this answers is not in the document]
Comment (tmsreekanth98) [reply amul_topicals.py]: i think we can ask the user at the end whether he/she would like to download more topicals and then loop only the part from where the user choice begins. We would not need to repeat the folder creating parts again.

**Comment by vineetjc** [comment]
Some bugs creeped up, check screenshots for reference:

scr3

scr2

scr1

**Comment by vineetjc** [comment]
Runs well, I'm merging it.

**Pull request merged**